scripts/us_bea/states_gdp/test_import.py

```diff
@@ -18,10 +18,12 @@
 
     python3 test_import.py
 """
+
 import unittest
+
 import pandas as pd
-import import_data
-import import_industry_data_and_gen_mcf
+from us_bea.states_gdp import import_data
+from us_bea.states_gdp import import_industry_data_and_gen_mcf
```

Comment:
I think these should be

import us_bea.states_gdp

and change all uses to "states_gdp.import_data" and "states_gdp.import_industry_data_and_gen_mcf".

Reply:
We should add this to the best practices notes as well since we are likely to have many top level scripts paths with bushy directory trees and not just one 1:1.

Reply:
Does https://google.github.io/styleguide/pyguide.html#22-imports suffice?